diagnostic.md

```diff
@@ -53,9 +69,23 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+```CSS
+.big {
+  ...
+}
 
-<!-- your answer ends here -->
+#contentPane {
+  ...
+}
+
+.important .red {
+  ...
+}
+
+div p {
```

Comment:
`div p` gets all `p` tags that are down the family tree of `div`, but to be more specific and get direct children of a selector, you'd use `div > p`